test/store.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { assert } from '.pnpm/@vue+compiler-core@3.2.29/node_modules/@vue/compiler-core'
+import { DeepReadonly } from 'vue'
+import { Store, Getters, Mutations, Module } from '../lib'
+
+interface TestState {
+  id: number
+  name: string
+}
+
+interface TestStateGetters extends Getters<TestState> {
+  isDefault: (state: DeepReadonly<TestState>) => boolean
+  upperCaseName: (state: DeepReadonly<TestState>) => string
+}
+
+interface TestStateMutations extends Mutations<TestState> {
+  setState: (state: TestState) => (newState: TestState) => void
+  updateName: (state: TestState) => (name: string) => void
+  clear: (state: TestState) => () => void
+}
+
+const getDefaultState = (): TestState => ({
+  id: 0,
+  name: '',
+})
+
+const makeTestModule = () =>
+  new Module<TestState, TestStateGetters, TestStateMutations>({
+    state: getDefaultState,
+    getters: {
+      isDefault: (state) => state.id !== 0 || state.name !== '',
```

Comment:
If I understand the meaning of `isDefault` correctly, the returned value of this function will actually be `notDefault`, right?

Reply:
Yes, good catch!